apps/ledger-live-desktop/src/renderer/modals/Send/fields/RecipientField.tsx

```diff
@@ -48,6 +48,21 @@ const RecipientField = <T extends Transaction, TS extends TransactionStatus>({
     }
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
+  useEffect(() => {
```

Comment:
wdyt about combine both useEffect ? With something like that?

```ts
useEffect(() => {
    if (value !== "" && value !== transaction.recipient) {
      onChangeTransaction(bridge.updateTransaction(transaction, { recipient: value }));
      if (resetInitValue && initValue) {
        resetInitValue();
      }
    }
  }, [
    value,
    transaction.recipient,
    bridge,
    onChangeTransaction,
    transaction,
    initValue,
    resetInitValue,
    account.id,
  ]);
```

Reply:
i did before, it didn't work.  `if (resetInitValue && initValue)` is not a valid condition to reset the initValue. Also the first effect will run one single time at the component mount. What i need is a second layer that manages the change of account.id basically. I would rather ignore exhaustive deps, but the functional works

Reply:
what you recommend as a refactor is a better abstraction, but with an uncontrolled resetInitiallValue, it gaves me a different behaviour.